crowbar_framework/app/models/ceph_service.rb

```diff
@@ -137,9 +137,9 @@ def create_proposal
     end
     mon_nodes = mon_nodes.take(mon_nodes.length > 2 ? 3 : 1)
 
-    mds_node = select_nodes_for_role(nodes, "ceph-mds", "controller").first
+    mds_node = select_nodes_for_role(nodes, "ceph-mds", "storage").first
 
-    radosgw_node = select_nodes_for_role(nodes, "ceph-radosgw", "controller").first
+    radosgw_node = select_nodes_for_role(nodes, "ceph-radosgw", "storage").first
```

Comment:
As an API service I would expect it to be fine running on the controller, and I would not expect it to cohabitate a host with the storage services. The docs here https://www.suse.com/documentation/ses-4/book_storage_admin/data/ceph_install_crowbar_deploy_nodes.html recommend installing it on a dedicated node, so I guess as long as this is separate from the ceph-mon and ceph-osd nodes it will be fine?